allensdk/brain_observatory/behavior/session_apis/data_io/behavior_ophys_lims_api.py

```diff
@@ -0,0 +1,164 @@
+import logging
+from typing import Optional
+import pandas as pd
+
+from allensdk.api.cache import memoize
+from allensdk.brain_observatory.behavior.session_apis.data_io.ophys_lims_api \
+    import OphysLimsApi
+from allensdk.brain_observatory.behavior.session_apis.data_io import (
+    BehaviorLimsApi)
+from allensdk.internal.api import db_connection_creator, PostgresQueryMixin
+from allensdk.internal.core.lims_utilities import safe_system_path
+from allensdk.core.auth_config import (
+    LIMS_DB_CREDENTIAL_MAP, MTRAIN_DB_CREDENTIAL_MAP)
+from allensdk.core.authentication import credential_injector, DbCredentials
+from allensdk.brain_observatory.behavior.session_apis.data_transforms import (
+    BehaviorOphysDataXforms)
+
+
+class BehaviorOphysLimsApi(BehaviorOphysDataXforms,  OphysLimsApi,
+                           BehaviorLimsApi):
+
+    def __init__(self, ophys_experiment_id: int,
+                 lims_credentials: Optional[DbCredentials] = None,
+                 mtrain_credentials: Optional[DbCredentials] = None):
+
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+        self.lims_db = db_connection_creator(
+            credentials=lims_credentials,
+            default_credentials=LIMS_DB_CREDENTIAL_MAP)
+
+        self.mtrain_db = db_connection_creator(
+            credentials=mtrain_credentials,
+            default_credentials=MTRAIN_DB_CREDENTIAL_MAP)
+
+        self.ophys_experiment_id = ophys_experiment_id
+        self.behavior_session_id = self.get_behavior_session_id()
+
+    def get_ophys_experiment_id(self):
+        return self.ophys_experiment_id
+
+    @memoize
+    def get_ophys_session_id(self):
+        query = '''
```

Comment:
nit: inconsistent query format in code relative to behavior_lims_api

Reply:
This should now be addressed.